Riot/Modules/profile/views/Dot.swift

```diff
@@ -0,0 +1,64 @@
+// 
+// Copyright 2020 Vector Creations Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+
+@IBDesignable class Dot: UIView {
+
+    @IBInspectable var mainColor: UIColor = UIColor.blue {
+        didSet {
+             print("mainColor was set here")
+        }
+    }
+
+    @IBInspectable var ringColor: UIColor = UIColor.orange {
+         didSet {
+             print("bColor was set here")
+        }
+    }
+
+    @IBInspectable var ringThickness: CGFloat = 4 {
+        didSet {
+            print("ringThickness was set here")
```

Comment:
Are these print messages shown to the user?

Reply:
These are console outputs only, visible in debug mode.  The Riot project does not have a Scheme sensitive debug function, so the use of `print()` is consistent with how the Riot team handle these type of information output functions (ref `EditHistoryViewModel.swift:97` et al).